From: Thomas Loimer <thomas.loimer@tuwien.ac.at>
Date: Tue Dec 22 20:03:07 2020 +0100
Subject: Include necessary header config.h in figx.h
Origin: https://sourceforge.net/p/mcj/xfig/ci/b34bce
Forwarded: via Mail to Thomas Loimer


--- a/src/figx.h
+++ b/src/figx.h
@@ -1,6 +1,9 @@
 /*
  * FIG : Facility for Interactive Generation of figures
- * Copyright (c) 1989-2007 by Brian V. Smith
+ * Copyright (c) 1985-1988 by Supoj Sutanthavibul
+ * Parts Copyright (c) 1989-2015 by Brian V. Smith
+ * Parts Copyright (c) 1991 by Paul King
+ * Parts Copyright (c) 2016-2020 by Thomas Loimer
  *
  * Any party obtaining a copy of these files is granted, free of charge, a
  * full and unrestricted irrevocable, world-wide, paid up, royalty-free,
@@ -16,6 +19,10 @@
 #ifndef FIGX_H
 #define FIGX_H
 
+#if defined HAVE_CONFIG_H && !defined VERSION
+#include "config.h"
+#endif
+
 #include <X11/cursorfont.h>
 #include <X11/StringDefs.h>
 #include <X11/Shell.h>
